lib/pages/movie_menu.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/widgets/menu/header.dart';
+import 'package:movie_card/shared/ui_consts.dart';
+import 'package:movie_card/shared/custom_card.dart';
+import 'package:movie_card/widgets/menu/menu_movie_details.dart';
+
+class MovieMenu extends StatefulWidget {
+  const MovieMenu({super.key});
+
+  @override
+  State<MovieMenu> createState() => _MovieMenuState();
+}
+
+class _MovieMenuState extends State<MovieMenu> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: COLORS.onPrimaryContainer,
+      body: Container(
+        decoration: const BoxDecoration(
+          image: DecorationImage(
+            image: AssetImage('assets/Menu_Background.png'),
+            fit: BoxFit.cover,
+          ),
+        ),
+        child: SingleChildScrollView(
+          child: Column(
+            children: [
+              const Header(),
+              Text(
+                "Lo último que viste",
+                style: SUBTITLE_STYLE,
+                textAlign: TextAlign.left,
+              ),
+              Column(
+                children: [
+                  Text(
+                    "Peliculas",
+                    style: SUBTITLE_STYLE,
+                  ),
+                  ...(List.generate(
+                      10,
+                      (index) => Padding(
+                            padding: const EdgeInsets.all(12),
+                            child: CustomCard(
+                              padding: 0,
+                              child: SizedBox(
+                                width: MediaQuery.of(context).size.width * 0.9,
+                                height:
+                                    MediaQuery.of(context).size.width * 0.45,
+                                child: MenuMovieDetails(
+                                  posterWidth:
+                                      MediaQuery.of(context).size.width *
+                                          0.45 *
+                                          0.6,
```

Comment:
You could extract this as a standalone widget. Also, use constants for the size multipliers here. 

To be entirely honest, I don't like using these kind of relative sizing, I prefer replacing them with well-placed Padding or spacers. But it's not bad to use them anyways.